skimage/data/__init__.py

```diff
@@ -1,3 +1,10 @@
+"""
+Image data module.
+
+Contains test images and example data.
```

Comment:
```suggestion
Images and datasets for tests and examples.

Makes a curated set of general purpose and scientific images and datasets easily
available for tests, examples and documentation. Newer datasets are no longer included
in scikit-image directly but are downloaded if accessed. To make them available offline,
 use  :func:`download_all`.
```

Reply:
I was not aware of the level of detail this module required. Thanks so much for the suggestion! I have incorporated it. I apologise for not creating a separate PR/branch for this, I made an error there.

Reply:
It's not required but I thought as we are already on it we can use the chance to highlight that some datasets need to be downloaded. :)